app-backend/dynamodb/handler.js

```diff
@@ -23,14 +23,20 @@ exports.graphqlHandler = function graphqlHandler(event, context, callback) {
 };
 
 // for local endpointURL is /graphql and for prod it is /stage/graphql
-exports.playgroundHandler = lambdaPlayground({
-  endpoint: process.env.REACT_APP_GRAPHQL_ENDPOINT
-    ? process.env.REACT_APP_GRAPHQL_ENDPOINT
-    : '/production/graphql',
-});
+exports.playgroundHandler = (event, context, callback) => {
```

Comment:
dont think we need this

Reply:
Need what?

Reply:
I mean I don't think we need these changes in rest and dynamodb backend. GraphiQL and Playground are working with those backends. We only see problem with RDS